torchrec/distributed/planner/proposers.py

```diff
@@ -279,6 +280,147 @@ def feedback(
         self._proposal_index += 1
 
 
+def _bytes_to_float_bin(num_bytes: Union[float, int], bin_size: float) -> float:
+    return float(num_bytes) / bin_size
+
+
+class DpProposer(Proposer):
```

Comment:
nit: rename to `DynamicProgrammingProposer` as `Dp` in torchrec usually refers to Data Parallel

```suggestion
class DynamicProgrammingProposer(Proposer):
```